libs/dh/market-participant/actors/feature-actors/src/lib/create/steps/dh-new-organization-step.component.ts

```diff
@@ -44,7 +44,8 @@ import { TranslocoDirective } from '@ngneat/transloco';
       :host {
         display: block;
       }
-
+      h4 {
+      }
```

Comment:
suggestion: maybe missing styling?

```suggestion
      h4 {
        margin-top: 0;
      }
```

Reply:
Undo gone wrong.. thanks for finding